src/assets/Generator.Shared/LowLevelFuncOperation.cs

```diff
@@ -44,6 +44,12 @@ internal LowLevelFuncOperation(ClientDiagnostics clientDiagnostics, HttpPipeline
         /// <inheritdoc />
         public override ValueTask<Response> UpdateStatusAsync(CancellationToken cancellationToken = default) => _operation.UpdateStatusAsync(cancellationToken);
 
+        /// <inheritdoc />
+        public override Response<T> WaitForCompletion(CancellationToken cancellationToken = default) => _operation.WaitForCompletion(cancellationToken);
+
+        /// <inheritdoc />
+        public override Response<T> WaitForCompletion(TimeSpan pollingInterval, CancellationToken cancellationToken = default) => _operation.WaitForCompletion(pollingInterval, cancellationToken);
+
```

Comment:
Sync version of WaitForCompletion method was not existed before. It is recently added in this [PR](https://github.com/Azure/azure-sdk-for-net/pull/22420/files#diff-d8fef9e5888142bd6d9a5398c2d61a54cde4d38e79fdd50a204650a1cf44abaaR139). It is needed in order to call `operation.WaitForCompletion();` for LLC LRO methods. [Here](https://github.com/Azure/autorest.csharp/blob/3c8aa4e060ca4729082a78bbd76ab43ade5facb2/test/TestProjects/LRO-LowLevel/Generated/LROServiceClient.cs#L112) is an example.